src/JLSOFile.jl

```diff
@@ -79,13 +95,29 @@ function Base.:(==)(a::JLSOFile, b::JLSOFile)
         a.version == b.version &&
         a.julia == b.julia &&
         a.image == b.image &&
-        a.pkgs == b.pkgs &&
+        a.manifest_toml == b.manifest_toml &&
         a.format == b.format &&
         a.compression == b.compression &&
         a.objects == b.objects
     )
 end
 
 Base.names(jlso::JLSOFile) = collect(keys(jlso.objects))
+function Base.getproperty(jlso::JLSOFile, attr::Symbol)
+    if attr === :pkgs
+        @warn "pkgs property is deprecated, use manifest or manifest_toml"
```

Comment:
`manifest` and `manifest_toml` return different things, right? I'm just wondering - if these are both public API - if we should document them somewhere?

Reply:
I'm not sure if we want `manifest_toml` to be part of the public API as we'll probably want to add support for explicitly printing/writing the toml rather than supporting people manipulating those strings directly.

Reply:
in which case let's just not mention `manfest_toml` in this message?